queries.sql

```diff
@@ -125,3 +125,110 @@ ON owner_id = owners.id
 GROUP BY full_name
 ORDER BY COUNT(owner_id) DESC
 LIMIT 1;
+
+--Who was the last animal seen by William Tatcher?
+SELECT vets.name AS vet_name,
+  animals.name AS animal_name,
+  visit_date AS last_visit
+FROM vets
+JOIN visits ON vets.id = vet_id
+JOIN animals ON animal_id = animals.id
+WHERE vets.name = 'William Tatcher'
+GROUP BY vets.name, animals.name, visit_date
+ORDER BY MAX(visit_date) DESC
+LIMIT 1;
```

Comment:
As per the screenshot you have attached for this query, it is returning the wrong results. The result of this query is returning `Agumon` which is not the last animal seen by William Tatcher. Kindly fix this issue.